Docs: quickstart (#21)
grafana/grafana-javascript-agent: README.md

> *Warning*: currently pre-release and subject to frequent breaking changes. Use at your own risk.

Reviewer: ```suggestion
*Warning*
This feature is currently pre-release and subject to frequent breaking changes.
```

> Grafana JavaScript Agent is a library that can instrument javascript applications

Reviewer 2: ```suggestion
Grafana JavaScript Agent is a library that can instrument JavaScript applications
```

(both frontend and (soon!) backend) to collect telemetry and forward it
to the [Grafana Agent](https://grafana.com/docs/agent/latest/)

> (with app o11y collector integration enabled). Grafana Agent then can send this data further to [Prometheus](https://prometheus.io/),

Reviewer 2: We should adopt that feature of Markdown where we set links in the footer of the document and the link them inline. This will allow us to point to the `Grafana Agent` (for example) page everywhere instead of one time only (as a developer I hate when i have to scroll to look for a clickable term 😢 ).

Author: oh, didn't know this was a thing
- [ ] footer links of the document

Reviewer: I think it's just a preference. I prefer inline links and that's what the docs team mostly uses as best practice, but always open to new ways :)

Reviewer: I think it's just a preference. I prefer inline links and that's what the docs team mostly uses as best practice, but always open to new ways :)

Author: So add normal link on every mention, or keep as is - just the first one, wdyt @brendamuir?

Reviewer: If it's multiple mentions in the same topic, just link from the first one

Author: ok, then lets keep as is

> Grafana JavaScript Agent is a library that can instrument JavaScript applications
> (both frontend and (soon!) backend) to collect telemetry and forward it
> to the [Grafana Agent](https://grafana.com/docs/agent/latest/)
> (with app o11y collector integration enabled). Grafana Agent then can send this data further to [Prometheus](https://prometheus.io/),

Reviewer: ```suggestion
Grafana JavaScript Agent is a library that can instrument frontend JavaScript applications to collect telemetry and forward it
to the [Grafana Agent](https://grafana.com/docs/agent/latest/)
(with app o11y collector integration enabled). We are also planning to implement this for backend JavaScript applications. Grafana Agent can then send this data to [Prometheus](https://prometheus.io/),
```

> [Loki](https://grafana.com/logs/), [Tempo](https://grafana.com/traces/).

Reviewer: ```suggestion
[Loki](https://grafana.com/logs/), or [Tempo](https://grafana.com/traces/).
```

> The repository consists of multiple packages that should be combined depending on the needs, as well as a [demo](https://github.com/grafana/grafana-javascript-agent/tree/main/demo)

Reviewer: ```suggestion
The repository consists of multiple packages that can be combined depending on your requirements, as well as a [demo](https://github.com/grafana/grafana-javascript-agent/tree/main/demo),
```

> which can be ran by following the [README.md file](https://github.com/grafana/grafana-javascript-agent/tree/main/demo/README.md).

Reviewer: ```suggestion
which can be run by following the [README.md file](https://github.com/grafana/grafana-javascript-agent/tree/main/demo/README.md).
```
